ingestion_tools/scripts/common/yaml_files.py

```diff
@@ -12,6 +12,7 @@
 
 
 def get_yaml_config_files(
+    input_files: list[str] = None,
     include_glob: str = None,
     exclude_keywords_list: list[str] = EXCLUDE_KEYWORDS_LIST,
     dataset_configs_dir: str = DATASET_CONFIGS_DIR,
```

Comment:
It's best to not use mutable default values in python function definitions. If you're *very* diligent to never mutate those values it's technically safe, but imop it's easier to avoid the situation entirely:
https://docs.python-guide.org/writing/gotchas/

Reply:
Adjusted!